flaskapi/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup, find_packages
+
+setup(name='opgflaskapi', 
```

Comment:
I suggest we put some linting on this. I would format this with each kwarg on its own line, indented 4 spaces:
```
setup(
    name='opgflaskapi',
    version='1.0', 
    description='base flask api',
    install_requires = [
        'aws_xray_sdk',
        'flask-sqlalchemy',
        'flask',
        'psycopg2-binary',
        'gunicorn',
    ],
    packages=[
        'opgflaskapi',
    ]
)
```

Reply:
Can you please raise an issue for that @nickdavis2001? Thanks.